loinc-integration/src/test/java/dev/ikm/tinkar/loinc/integration/LoincDefinitionSemanticIT.java

```diff
@@ -0,0 +1,232 @@
+		if (!longCommonName.isEmpty()) {
+			descType = FULLY_QUALIFIED_NAME_DESCRIPTION_TYPE;
+			term = longCommonName;
+			
+			termConceptMap.put(getConceptMapKey(concept, term), getConceptMapValue(descType, term));
+		}
+
+		if (!consumerName.isEmpty()) {
+			descType = REGULAR_NAME_DESCRIPTION_TYPE;
+			term = consumerName;
+			
+			termConceptMap.put(getConceptMapKey(concept, term), getConceptMapValue(descType, term));
+		}
+
+		if (!shortName.isEmpty()) {
+			descType = REGULAR_NAME_DESCRIPTION_TYPE;
+			term = shortName;
+			
+			termConceptMap.put(getConceptMapKey(concept, term), getConceptMapValue(descType, term));
+		}
+
+		if (!relatedNames2.isEmpty()) {
+			descType = REGULAR_NAME_DESCRIPTION_TYPE;
+			term = relatedNames2;
+			
+			termConceptMap.put(getConceptMapKey(concept, term), getConceptMapValue(descType, term));
+		}
+
+		if (!displayName.isEmpty()) {
+			descType = REGULAR_NAME_DESCRIPTION_TYPE;
+			term = displayName;
+			
+			termConceptMap.put(getConceptMapKey(concept, term), getConceptMapValue(descType, term));
+		}
+
+		if (!definitionDescription.isEmpty()) {
+			descType = DEFINITION_DESCRIPTION_TYPE;
+			term = definitionDescription;
+			
+			termConceptMap.put(getConceptMapKey(concept, term), getConceptMapValue(descType, term));
+		}
+
+		StateSet active = null;
+		if (columns[11].equals("ACTIVE") || columns[11].equals("TRIAL") || columns[11].equals("DISCOURAGED")) {
+			active = StateSet.ACTIVE;
+		} else {
+			active = StateSet.INACTIVE;
+		}
+
+		AtomicBoolean matched = new AtomicBoolean(false);
+		
+		StampCalculator stampCalc = StampCalculatorWithCache
+				.getCalculator(StampCoordinateRecord.make(active, Coordinates.Position.LatestOnMaster()));
+		
+		PatternEntityVersion latestDescriptionPattern = (PatternEntityVersion) Calculators.Stamp.DevelopmentLatest()
+				.latest(TinkarTerm.DESCRIPTION_PATTERN).get();
+		
+		EntityService.get().forEachSemanticForComponentOfPattern(concept.nid(), TinkarTerm.DESCRIPTION_PATTERN.nid(), semanticEntity -> {
+			count++;
+			innerCount.incrementAndGet();
+			
+			Latest<SemanticEntityVersion> latest = stampCalc.latest(semanticEntity);
+			UUID semanticEntityUUID = semanticEntity.asUuidArray()[0];
+			
+			ConceptMapValue cmv = termConceptMap.get(semanticEntityUUID);
+						
+			if(cmv != null) {
+				if (latest.isPresent()) {
+					Component descriptionType = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.DESCRIPTION_TYPE,
+							latest.get());
+					
+					Component caseSensitivity = latestDescriptionPattern
+							.getFieldWithMeaning(TinkarTerm.DESCRIPTION_CASE_SIGNIFICANCE, latest.get());
+					
+					String text = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.TEXT_FOR_DESCRIPTION, latest.get());
+					
+					//if (PublicId.equals(semanticEntity.publicId(), TinkarTerm.DEFINITION_DESCRIPTION_TYPE.publicId())) {
+						
+					if (descriptionType.equals(cmv.conceptDescType) && caseSensitivity.equals(DESCRIPTION_NOT_CASE_SENSITIVE) 
```

Comment:
This if statement should be the opposite of what it is currently, so something like (if !... || !... || !....) {
matched.set(false);